app/redux/cartItems.js

```diff
@@ -58,18 +58,18 @@ export const cartItemsReducer = (state = [], action) => {
 
 //thunks
 
-export const setCartItems = (cartId) => {
+export const fetchCartItems = (cartId) => {
   return (dispatch) => {
```

Comment:
you can actually just write this as 
`return (dispatch, getState, {axios}) =>`
for each thunk and not have to import axios into any of the redux files

Reply:
👍